sqlite_fdw.c

```diff
@@ -1114,12 +1114,14 @@ sqliteGetForeignPaths(PlannerInfo *root, RelOptInfo *baserel, Oid foreigntableid
 		ParamPathInfo *param_info = (ParamPathInfo *) lfirst(lc);
 		double		rows;
 		int			width;
+		Cost		startup_cost1;
+		Cost		total_cost1;
```

Comment:
Changed in https://github.com/pgspider/sqlite_fdw/pull/88/commits/08d87482359053057ff47d4418fbd265069a5a25 This will be really better names, Thanks @nxhai98 !

Reply:
Thank you. Please help me to resolve this thread.